src/Auth0/Login/Auth0Service.php

```diff
@@ -51,10 +51,10 @@ public function logout()
     /**
      * Redirects the user to the hosted login page
      */
-    public function login($connection = null, $state = null, $aditional_params = ['scope' => 'openid profile email'], $response_type = 'code')
+    public function login($connection = null, $state = null, $additional_params = ['scope' => 'openid profile email'], $response_type = 'code')
```

Comment:
This looks at first like a *BREAKING CHANGE*. Is it just the parameter name, used internally in this method logic or can people actually call this method passing a dictionary with the `${param_name}` as keys?

Reply:
@lbalmaceda - No fancy stuff here, just the internal signature.